src/recipe/test/e2e/findRecipeById.controller.e2e-spec.ts

```diff
@@ -25,8 +28,10 @@ describe('Recipe Controller - Find By Id', () => {
 
   beforeAll(async () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [RecipeModule, AuthModule],
+      imports: [RecipeModule, AuthModule, AppModule],
```

Comment:
I assume that AppModule imports RecipeModule, AuthModule
Do you need to import Recipe and Auth modules separately?